.github/actions/setup-env-vars/action.yml

```diff
@@ -0,0 +1,33 @@
+name: "Setup Env Vars"
```

Comment:
I think you may consider to rebase against the latest master. These files should not appear I think.

Reply:
Yes, please do so to make it easier to review

Reply:
@sxjscience @Innixma I rebased and it seems that this file is no longer there.  I also added unitttests and finished detector - should be good to review